### ISSUE TITLE
Enhance multiple avatar display

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/android#6019]
For the Android Deck app, @stefan-niedermann added a very nice 2dp border as per https://github.com/stefan-niedermann/nextcloud-deck/issues/433

Would be very nice to use this in the file list as well, and possibly in other places where we show multiple avatars:

> ![grafik](https://user-images.githubusercontent.com/4741199/80514584-01128980-8981-11ea-8685-4c8c415e1d95.png)
> ![grafik](https://user-images.githubusercontent.com/4741199/80514628-0d96e200-8981-11ea-8309-099c44d0a834.png)

@stefan-niedermann also said:

> In theory this is reusable.
> 
> @tobiasKaminsky if you want to reuse this component, i will put some heart into it and extract it into a own module package.
> 
> But beware, the border is a hard color value which has to be set depending on where you are using the OverlappingAvatarsLayout - because in the sample case the background of the material card uses a transparent color, and if use the exact same color it will multiply and the the border-color will be darker than the background of the cards.
> 
> TL;DR: Contact me in Telegram if you want to reuse it.

**Comment by asaikarthikeya** [comment]
I want to work on this issue. Can you guide me how to get started with this.

**Comment by stefan-niedermann** [comment]
@asaikarthikeya Maybe you can copy & paste some stuff from the [`OverlappingAvatars` component](https://github.com/stefan-niedermann/nextcloud-deck/blob/master/app/src/main/java/it/niedermann/nextcloud/deck/ui/view/OverlappingAvatars.java) of the Deck android app 🙂.

**Comment by AndyScherzinger** [comment]
@asaikarthikeya that be great, so, yes please ❤️

**Comment by asaikarthikeya** [comment]
@stefan-niedermann @AndyScherzinger 
I am unable to find file list and other places where multiple avatars are used.

Kindly, Help me in understand this issue better.
Thank you in advance.

**Comment by tobiasKaminsky** [comment]
Here we loop over sharees and add them to itemViewHolder.sharedAvatars.

https://github.com/nextcloud/android/blob/149dce75715a0152ed57c7998a4c89f50fba58a5/src/main/java/com/owncloud/android/ui/adapter/OCFileListAdapter.java#L428

**Comment by AndyScherzinger** [comment]
PR implementing this based on or (heavily) inspired by @stefan-niedermann's UI component:  #8137
